src/ray/protobuf/gcs.proto

```diff
@@ -690,4 +665,32 @@ message PlacementGroupTableData {
   // effectively when the placement group has been "scheduled".
   int64 placement_group_final_bundle_placement_timestamp_ms = 15;
 }
+
+message JobTableData {
+  // The job ID.
+  bytes job_id = 1;
+  // Whether it's dead.
+  bool is_dead = 2;
+  // The UNIX timestamp corresponding to this event (job added or removed).
+  int64 timestamp = 3;
+  // IP address of the driver that started this job.
+  // Deprecated
```

Comment:
How about this : https://protobuf.dev/programming-guides/proto3/#options

```
string driver_ip_address = 4 [deprecated = true];
```

Reply:
Good idea, looks like this only adds a warning at compilation time, so there shouldn't be any ill effects for the user

Reply:
ah, seems we treat warnings as errors at compilation.

Reply:
ah but actually, why are we still setting `driver_ip_address` if this is being deprecated?

Reply:
@rickyyx Good question. It turns out it's only called in a util function which is only called in test code.  There must be a way to work around this, because I'm guessing it's common to test deprecated code until it's actually deleted.  But if I can't figure it out, I'll just delete it from the test code.

Reply:
(Or just mark it with comments)

Reply:
Since it's a time sensitive release blocker I'll just mark it with comments for now.